contributing/reviewing-prs.md

```diff
@@ -17,7 +17,7 @@ This guide is for all committers and contributors that want to help with reviewi
 
 ## Review Checklist
```

Comment:
I think points 1, 2, 3 and 4 of the review process should be moved to another page (or removed) as these discussions should happen in Jira and not the PR. A good, concise description is still required but the discussions should be completed when the PR is opened. Hence, the PR review process can be significantly simplified to check if description is OK, check that the PR implements what was proposed, check code quality of the PR.

Reply:
I reworded these sections. Let's further iterate on these in another PR.